src/main/java/nl/tudelft/contextproject/model/Game.java

```diff
@@ -89,4 +104,35 @@ public Level getLevel() {
 	public void setLevel(Level level) {
 		this.level = level;
 	}
+	
+	/**
+	 * End this game.
+	 * 
+	 * @param didElvesWin
+	 * 		true when the elves won, false when the dwarfs did
+	 */
+	public void endGame(boolean didElvesWin) {
+		controller.gameEnded(didElvesWin);
+	}
+
+	/**
+	 * Update the game timer.
+	 * 
+	 * @param tpf
+	 * 		the time per frame of this update
+	 */
+	public void update(float tpf) {
+		timeLimit -= tpf;
```

Comment:
We are the worst game developer if we bind time to the time per frame. I suggest we change it to use System.currentTimeMillis() and a difference check instead. So that a inconsistent frame rate (likely with VR) has no impact on the length of a game!